src/webgui.js

```diff
@@ -26,6 +29,15 @@ app.get('/version', (req, res) => {
   res.status(200).sendFile(path.join(__dirname, 'public/version.pug'));
 });
 
+function refreshSession(req, res) {
+  if ( !req.cookies.discord_session || !req.cookies.discord_session.at) {
+    accessLog.info(`Invalid session, attempting refresh`);
+    res.redirect(`/api/discord/refresh?callback=/play/${req.params.clip}`);
+    return false;
```

Comment:
This return won't do anything, since we are redirecting above.

Reply:
Actually, it does. Since res.redirect is an async call, this function returns back to the caller with true/false, and then THEY decide if the processing is finished. I.E. for the index page.